src/readii/io/loaders/general.py

```
@@ -0,0 +1,78 @@
+
+    # Check if config file exists
+    if not os.path.exists(config_file_path):
+        raise FileNotFoundError(f"Config file {config_file_path} does not exist.")
+    
+    try:
+        # Load the config file
+        with open(config_file_path, "r") as f:
+            return yaml.safe_load(f)
+
+    except yaml.YAMLError as e:
+        raise ValueError(f"Invalid YAML in config file: {e}")  
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)